packages/api/src/schema/api-schema.yaml

```diff
@@ -479,6 +479,9 @@ components:
                     Longitude of the pull source in degrees. East is positive,
                     west is negative.
                   example: -104.988
+            lockedAt:
```

Comment:
I think ideally this field should be hidden from responses. In other resources this would just mean setting it as a `writeOnly: true` here, but for streams we have a separate clean-up function that handles removing fields from some const in the code: https://github.com/livepeer/livepeer-com/blob/ff6c3a0453d2a0cce8f9ace8649c8c1bbfac7a7a/packages/api/src/store/stream-table.ts#L390

That function doesn't support nested properties today tho, so I'll leave it up to you if it's worth the work of making it work with this nested `pull.lockedAt` field.

Reply:
Good point. I've moved `lockedAt` to non-nested `pullLockedAt` and prevent returning it from the API response.